products/storage/terraform.yaml

```diff
@@ -102,6 +102,32 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         ignore_read: true
   Object: !ruby/object:Overrides::Terraform::ResourceOverride
     exclude: true
+  HmacKey: !ruby/object:Overrides::Terraform::ResourceOverride
+    create_url: projects/{{project}}/hmacKeys?serviceAccountEmail={{service_account_email}}
```

Comment:
Can you add `skip_sweeper: true` with a comment above it that `# This resource does not have a name field`?